faq/faq.rst

```diff
@@ -12,13 +12,20 @@ Currently Citus imposes primary key constraint only if the distribution column i
 How do I add nodes to an existing Citus cluster?
 ------------------------------------------------
 
-You can add nodes to a Citus cluster by calling the master_add_node UDF with the hostname (or IP address) and port number of the new node. After adding a node to an existing cluster, it will not contain any data (shards). Citus will start assigning any newly created shards to this node. To rebalance existing shards from the older nodes to the new node, the Citus Enterprise edition provides a shard rebalancer utility. You can find more information about shard rebalancing in the :ref:`cluster_management` section.
+On Citus Cloud it's as easy as dragging a slider in the user interface. The :ref:`scaling_out` section has instructions. In Citus Community edition you can add nodes manually by calling the :ref:`master_add_node` UDF with the hostname (or IP address) and port number of the new node.
+
+Either way, after adding a node to an existing cluster it will not contain any data (shards). Citus will start assigning any newly created shards to this node. To rebalance existing shards from the older nodes to the new node, Citus Cloud and Enterprise edition provide a shard rebalancer utility. You can find more information in the :ref:`shard_rebalancing` section.
 
 How does Citus handle failure of a worker node?
 -----------------------------------------------
 
-If a worker node fails during e.g. a SELECT query, jobs involving shards from that server will automatically fail over to replica shards located on healthy hosts. This means intermittent failures will not require restarting potentially long-running analytical queries, so long as the shards involved can be reached on other healthy hosts.
-You can find more information about Citus' failure handling logic in :ref:`dealing_with_node_failures`.
+Using one of two methods depending on configuration:
+
+**Streaming replication** maintains a standby node for every worker, allowing a seamless failover to the standby on failure. Citus Community edition is compatible with streaming replication but does not automatically configure it on a local cluster, whereas Citus Cloud has streaming replication built in.
```

Comment:
I think https://docs.citusdata.com/en/v7.1/admin_guide/cluster_management.html#worker-node-failures captures it well. 
- I think we do want to note that Shard replication is better for append-only workloads.
- Instead of "compatible with", we should note that you can just use the built-in PostgreSQL streaming replication. 
Could you update this with content from that link? It probably captures the same amount of information, but worth noting here as well.